pyincore/analyses/epfdamage/epfdamage.py

```diff
@@ -256,7 +256,8 @@ def epf_damage_analysis_bulk_input(self, epfs, hazard_type, hazard_dataset_id,
                 liquefaction_prob = liquefaction_vals[i]['liqProbability']
 
                 selected_liq_fragility = liq_fragility_set[liq_epf_id]
-                pgd_limit_states = selected_liq_fragility.calculate_limit_state(liq_hazard_val, std_dev=std_dev)
```

Comment:
Related to this, EPF damage's liquefaction code is also using the below method that is deprecated and returns damage states in the format of "insignific", "moderate" etc. that are not used anymore.
https://github.com/IN-CORE/pyincore/blob/develop/pyincore/utils/analysisutil.py#L74

We should also target removing this `AnalysisUtil.calculate_damage_interval()`. Feel free to create a new issue to address this, if it's too complex to include in this PR.

This is being used in line 276 of this file. I couldn't figure out how to add a comment on that line, so please excuse the placement of this comment